src/templates/use-cases.tsx

```diff
@@ -25,135 +39,66 @@ import Logo from "../components/Logo"
-    padding-top: 4rem;
-  }
-  @media (max-width: ${(props) => props.theme.breakpoints.l}) {
-    flex-direction: column;
-  }
-`
-
-const InfoColumn = styled.aside`
-  display: flex;
-  flex-direction: column;
-  position: sticky;
-  top: 6.25rem; /* account for navbar */
-  height: calc(100vh - 80px);
-  flex: 0 1 400px;
-  margin-right: 4rem;
-  margin-left: 2rem;
-  @media (max-width: ${(props) => props.theme.breakpoints.l}) {
-    display: none;
-  }
-`
-
-const MobileButton = styled.div`
-  @media (max-width: ${(props) => props.theme.breakpoints.l}) {
-    background: ${(props) => props.theme.colors.background};
-    box-shadow: 0 -1px 0px ${(props) => props.theme.colors.border};
-    width: 100%;
-    bottom: 0;
-    position: sticky;
-    padding: 2rem;
-    z-index: 99;
-    margin-bottom: 0rem;
-  }
-`
+import { ChildOnlyProp, Context } from "../types"
 
-// Apply styles for classes within markdown here
-const ContentContainer = styled.article`
-  flex: 1 1 ${(props) => props.theme.breakpoints.l};
-  position: relative;
-  padding: 2rem;
-  padding-top: 0rem;
-
-  .featured {
-    padding-left: 1rem;
-    margin-left: -1rem;
-    border-left: 1px dotted ${(props) => props.theme.colors.primary};
-  }
-
-  .citation {
-    p {
-      color: ${(props) => props.theme.colors.text200};
-    }
-  }
-`
+const commonHeadingProps: HeadingProps = {
+  fontWeight: 700,
+  lineHeight: 1.4,
+}
 
-const Pre = styled.pre`
-  max-width: 100%;
-  overflow-x: scroll;
-  background-color: ${(props) => props.theme.colors.preBackground};
-  border-radius: 0.25rem;
-  padding: 1rem;
-  border: 1px solid ${(props) => props.theme.colors.preBorder};
-  white-space: pre-wrap;
-`
+const H1 = (props: HeadingProps) => (
+  <Heading as="h1" {...commonHeadingProps} fontSize="2.5rem" {...props} />
+)
 
-const InfoTitle = styled.h2`
-  font-size: 3rem;
-  font-weight: 700;
-  text-align: right;
-  margin-top: 0rem;
-  @media (max-width: ${(props) => props.theme.breakpoints.l}) {
-    text-align: left;
-    font-size: 2.5rem;
-    display: none;
-  }
-`
+const H2 = (props: HeadingProps) => (
+  <Heading {...commonHeadingProps} fontSize="2rem" {...props} />
+)
 
-const H2 = styled.h2`
-  font-size: 2rem;
-  font-weight: 700;
-  margin-top: 4rem;
-`
+const H3 = (props: HeadingProps) => (
+  <Heading as="h3" {...commonHeadingProps} fontSize="2xl" {...props} />
+)
 
-const H3 = styled.h3`
-  font-size: 1.5rem;
-  font-weight: 700;
-`
+const H4 = (props: HeadingProps) => (
+  <Heading
+    as="h4"
+    {...commonHeadingProps}
+    fontSize="xl"
+    fontWeight={600}
+    {...props}
+  />
+)
 
 // Note: you must pass components to MDXProvider in order to render them in markdown files
 // https://www.gatsbyjs.com/plugins/gatsby-plugin-mdx/#mdxprovider
 const components = {
   a: Link,
-  h1: Header1,
+  h1: H1,
   h2: H2,
   h3: H3,
-  h4: Header4,
+  h4: H4,
   p: Paragraph,
   li: ListItem,
-  pre: Pre,
+  // pre: Pre,
```

Comment:
Isn't this used?

Reply:
Whoops! Not sure what I did here. Adding it back in. 😅